### PR TITLE
Rebalances OT rockets to be more incendiary rather than explosive

### PR DESCRIPTION
**Pull request by iloveloopers**
# About the pull request
OT rockets explosion power and falloff are now 240 power and 90 falloff
max container volume increased to 210 units (1 beaker, 1 bottle, 1 vial)
max fire intensity increased to 40 (same as blueflame)
max fire duration increased to 48 (max firestacks, blueflame is 40)
OT rockets can no longer make a star shaped fire
max range increased to 7 tiles
these changes will need to be edited in on the wiki
this should probably be test merged and numbers tweaked

# Explain why it's good for the game
OT rockets one shotting tier 3s is unfun for the receiving xeno, this PR alleviates that by reducing the maximum explosive power to that of the OT M15 grenade.
To counterbalance, maximum fire intensity and fire duration has been increased to allow for more creativity in mixes, increased max range of the rocket so the user doesn't get hit by his own flames and the fire can no longer be a star shape so that it can be effective.
You cannot max both incendiary and explosive parts without the help of custom research chemicals, the OT must make a choice or compromise.
OT rockets will still be a tier 2 (and marine) killer but overrall less powerful than before,

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

normal WP for reference:

https://github.com/cmss13-devs/cmss13/assets/140007537/2e14357a-1937-4729-b4cb-117241c879d5

a max incendiary rocket:

https://github.com/cmss13-devs/cmss13/assets/140007537/ae6e6ab3-bf1e-4cab-a744-e749478b54e6

</details>

# Changelog
:cl:Tim Harley cookpet
balance: OT rockets explosion power and falloff nerfed to be slightly more explosive than the OT M15 grenade (240 exp, 90 falloff).
balance: OT rockets max container volume increased to 210 units (1 beaker, 1 bottle, 1 vial).
balance: OT rockets max fire intensity increased to 40 and max duration increased to 48.
balance: OT rockets can no longer make a star shaped fire.
balance: OT rockets max range increased to 7 tiles.
/:cl:

### CODE REVIEW

File: code/game/objects/items/explosives/explosive.dm [hunk 1]
```diff
@@ -24,6 +24,7 @@
 									"min_fire_rad" = 1, "min_fire_int" = 3, "min_fire_dur" = 3
 	)
 	var/falloff_mode = EXPLOSION_FALLOFF_SHAPE_LINEAR
+	var/no_star_shape = FALSE //forces the shape of the flame to be normal if true
```
Comment (Drulikar) [inline code/game/objects/items/explosives/explosive.dm:27]:
```suggestion
	/// Whether a star shape is possible when the intensity meets CHEM_FIRE_STAR_THRESHOLD
	var/allow_star_shape = TRUE
```

File: code/game/objects/items/explosives/explosive.dm [hunk 2]
```diff
@@ -185,6 +186,7 @@
 	for(var/obj/item/reagent_container/glass/G in containers)
 		if(G.reagents.total_volume)
 			has_reagents = 1
+			reagents.no_star_shape = no_star_shape
```
Comment (Drulikar) [inline code/game/objects/items/explosives/explosive.dm:189]:
```suggestion
			reagents.allow_star_shape = allow_star_shape
```

File: code/game/objects/items/explosives/warhead.dm [hunk 3]
```diff
@@ -9,10 +9,11 @@
 	name = "84mm rocket warhead"
 	desc = "A custom warhead meant for 84mm rocket shells."
 	icon_state = "warhead_rocket"
-	max_container_volume = 180
+	max_container_volume = 210
+	no_star_shape = TRUE
```
Comment (Drulikar) [inline code/game/objects/items/explosives/warhead.dm:13]:
```suggestion
	allow_star_shape = FALSE
```

File: code/modules/reagents/Chemistry-Holder.dm [hunk 4]
```diff
@@ -4,6 +4,7 @@
 	var/maximum_volume = 100
 	var/atom/my_atom = null
 	var/trigger_volatiles = FALSE
+	var/no_star_shape = FALSE
```
Comment (Drulikar) [inline code/modules/reagents/Chemistry-Holder.dm:7]:
```suggestion
	var/allow_star_shape = TRUE
```

File: code/modules/reagents/Chemistry-Holder.dm [hunk 5]
```diff
@@ -678,7 +679,7 @@
 		duration = max_fire_dur
 
 	// shape
-	if(supplemented > 0 && intensity > CHEM_FIRE_STAR_THRESHOLD)
+	if(supplemented > 0 && intensity > CHEM_FIRE_STAR_THRESHOLD && src.no_star_shape == FALSE)
```
Comment (Drulikar) [inline code/modules/reagents/Chemistry-Holder.dm:682]:
```suggestion
	if(supplemented > 0 && intensity > CHEM_FIRE_STAR_THRESHOLD && allow_star_shape)
```

File: code/game/objects/items/explosives/warhead.dm [hunk 6]
```diff
@@ -9,10 +9,11 @@
...
 	matter = list("metal" = 11250) //3 sheets
-	reaction_limits = list( "max_ex_power" = 300, "base_ex_falloff" = 120,"max_ex_shards" = 64,
-							"max_fire_rad" = 7, "max_fire_int" = 30, "max_fire_dur" = 36,
+	reaction_limits = list( "max_ex_power" = 215, "base_ex_falloff" = 90,"max_ex_shards" = 64,
```
Comment (Drulikar) [inline code/game/objects/items/explosives/warhead.dm:15]: Likely the max_ex_power is nerfed too much, but I don't know for sure.

File: code/game/objects/items/explosives/warhead.dm [hunk 7]
```diff
@@ -9,10 +9,11 @@
...
+							"max_fire_rad" = 7, "max_fire_int" = 40, "max_fire_dur" = 48,
```
Comment (Drulikar) [inline code/game/objects/items/explosives/warhead.dm:16]:
```suggestion
							"max_fire_rad" = 6, "max_fire_int" = 40, "max_fire_dur" = 48,
```
Not really interested in the spec killing themselves even if they shoot at max range.

**Review by Drulikar — CHANGES_REQUESTED** [review]

File: code/game/objects/items/explosives/warhead.dm (hunk 6 above)
Comment (iloveloopers) [reply code/game/objects/items/explosives/warhead.dm]: increased to 240

**Review by Drulikar — APPROVED** [review]

**Pull request merged**